paper-radio-button.html

```diff
@@ -62,22 +65,25 @@
       }
 
       #radioContainer {
-        display: inline-block;
+        @apply(--layout-inline);
+        @apply(--layout-center-center);
         position: relative;
-        width: 16px;
-        height: 16px;
+        width: var(--paper-radio-button-size, 16px);
+        height: var(--paper-radio-button-size, 16px);
         vertical-align: middle;
       }
 
       #ink {
         position: absolute;
-        top: -16px;
```

Comment:
You need to update the `dir="rtl"` bit below this as well, otherwise in RT it looks like this:
<img width="147" alt="screen shot 2015-12-17 at 10 58 45 am" src="https://cloud.githubusercontent.com/assets/1369170/11878917/40923b72-a4ad-11e5-9bd1-c2be4d3f00f6.png">